command/command_test.go

```diff
@@ -240,7 +240,7 @@ var _ = Describe("BackgroundCmd", func() {
 				SetupMockExpect(cmd, manager, proc, syscall.SIGCONT, findErr, signalErr, times)
 
 				sut.KeepAlive()
-				sut.KeepAlive() // we call it twice volountarily to ensure a single goroutine is created
+				//sut.KeepAlive() // we call it twice voluntarily to ensure a single goroutine is created
```

Comment:
any idea why we used to call it twice? I don't see how that ensured that only one goroutine is created

Reply:
I mean we don't verify if two goroutines are created, but if another goroutines is created, methods of the processManager will not match the call expectations.